lib/util/DocumentExtractor.js

```diff
@@ -42,9 +42,31 @@ const extractors = ([
     methods: {
       extractFromRequest: request => {
         const documents = [];
-
-        for (let it = 0; it < request.input.body.ids.length; it++) {
-          documents.push({ _id: request.input.body.ids[it] });
+        if ( request.input.body
+          && request.input.body.ids
+          && Object.keys(request.input.body.ids).length
+        ) {
+          for (let it = 0; it < request.input.body.ids.length; it++) {
+            documents.push({ _id: request.input.body.ids[it] });
+          }
+        }
+        else if (request.input.args.ids) {
+          if (Array.isArray(request.input.args.ids)) {
+            documents.push({ _id: request.input.args.ids });
```

Comment:
Here you are pushing an entire array in the document `_id` property.
Also, there is no unit test associated to this case